Importer/Log/CsvLog.php

```diff
@@ -50,9 +50,12 @@ public function addDataError($index, $errorMessage)
     /**
      * {@inheritdoc}
      */
-    public function addCriticalError($errorMessage)
+    public function addCriticalError($message, $userError = false)
```

Comment:
What do you think about taking a bit different approach: instead of using boolean flag to switch between different "modes" (user/non-user), we could require 2 parameters: `$errorReason` and `$userMessage`.
Then, for each error that has non-empty `$userMassage` we could print it on GUI, errors without `$userMessages` would be just logged.